app/_src/quickstart/universal-demo.md

```diff
@@ -0,0 +1,185 @@
+
+{% endtab %}
+{% tab usage kumactl (Read/Write) %}
+
+You can use the `kumactl` CLI to perform **read-only** operations on {{site.mesh_product_name}} resources. The `kumactl` binary is a client to the {{site.mesh_product_name}} HTTP API, you will need to first port-forward the API service with:
+
+Run `kumactl`, for example:
+
+```sh
+kumactl get dataplanes
+# MESH      NAME                                              TAGS
+# default   kuma-demo-app-68758d8d5d-dddvg.kuma-demo          app=kuma-demo-demo-app env=prod pod-template-hash=68758d8d5d protocol=http service=demo-app_kuma-demo_svc_5000 version=v8
+# default   redis-master-657c58c859-5wkb4.kuma-demo           app=redis pod-template-hash=657c58c859 protocol=tcp role=master service=redis_kuma-demo_svc_6379 tier=backend
+```
+
+You can configure `kumactl` to point to any zone `kuma-cp` instance by running:
+
+```sh
+kumactl config control-planes add --name=XYZ --address=http://{address-to-kuma}:5681
+```
+{% endtab %}
+{% endtabs %}
+
+## Introduction to zero-trust security
+
+By default, the network is insecure and not encrypted. We can change this with {{site.mesh_product_name}} by enabling the [Mutual TLS](/docs/{{ page.version }}/policies/mutual-tls/) policy to provision a Certificate Authority (CA) that will automatically assign TLS certificates to our services (more specifically to the injected data plane proxies running alongside the services).
+
+Before enabling [Mutual TLS](/docs/{{ page.version }}/policies/mutual-tls/) (mTLS) in your mesh, you need to create a `MeshTrafficPermission` policy that allows traffic between your applications.
+
+{% warning %}
+If you enable [mTLS](/docs/{{ page.version }}/policies/mutual-tls/) without a `MeshTrafficPermission` policy, all traffic between your applications will be blocked. 
+{% endwarning %}
+
+To create a `MeshTrafficPermission` policy with a builtin CA backend, do the following:
+
+```sh
+echo 'type: Mesh
+	name: default
+	mtls:
+	  enabledBackend: ca-1
+	  backends:
+	  - name: ca-1
+	    type: builtin' | kumactl apply -f -
+```
+
+Once Mutual TLS has been enabled, {{site.mesh_product_name}} will **not allow** traffic to flow freely across our services unless we explicitly have a [Traffic Permission](/docs/{{ page.version }}/policies/traffic-permissions/) policy that describes what services can be consumed by other services.
+By default, a very permissive traffic permission is created.
+
+For the sake of this demo, we will delete it:
+
+```sh
+kumactl delete traffic-permission allow-all-default
+```
+
+You can try to make requests to the demo application at [`127.0.0.1:5000/`](http://127.0.0.1:5000/) and you will notice that they will **not** work.
+
+Now, let's add back the default traffic permission:
+```sh
+echo 'type: MeshTrafficPermission
+name: allow-all
+mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+  from:
+  - targetRef:
+      kind: Mesh
+    default:
+      action: Allow' | kumactl apply -f -
```

Comment:
@slonka I modified this one so it's similar to the edit you suggested elsewhere. Does this look correct?